common/lib/capa/capa/templates/codeinput.html

```diff
@@ -1,12 +1,16 @@
+<%page expression_filter="h"/>
 <%!
 from django.utils.translation import ugettext as _
 from openedx.core.djangolib.markup import HTML
 %>
-<section id="textbox_${id}" class="capa_inputtype textbox cminput">
-  <textarea rows="${rows}" cols="${cols}" name="input_${id}" 
-            aria-label="${_("{programming_language} editor").format(programming_language=mode)}" 
-            aria-describedby="answer_${id}" 
-            id="input_${id}" 
+<div id="textbox_${id}" class="capa_inputtype textbox cminput">
```

Comment:
@muzaffaryousaf Thanks for asking the question. It currently renders like this:

```
<section class="wrapper-problem-response" tabindex="-1" aria-labelledby="python_grader-problem-title python_grader_1-question-index">
<h4 id="python_grader_1-question-index" class="sr">Question 1</h4>
<div id="textbox_python_grader_2_1" class="capa_inputtype textbox cminput">
``` 

Once we wrap up https://github.com/edx/edx-platform/pull/13550 the parent of the div will be a div with role=group (and an aria-label). I don't want to add another group since some screen readers require an additional keystroke to interact with the group.  If we nest too many groups together it becomes cumbersome to use.  Groups are good when used sparingly.  

Does that make sense, or might I be missing something?  When both merge, it should look something like this which is good:

```
<div role="group" aria-label="Question 1" class="wrapper-problem-response" tabindex="-1">
<div id="textbox_python_grader_2_1" class="capa_inputtype textbox cminput">
```